core/shared/src/main/scala/zio/prelude/Traversable.scala

```diff
@@ -122,6 +122,24 @@ trait Traversable[F[+_]] extends Covariant[F] {
       }
     }
 
+  def forany[G[+ _] : AssociativeEither : IdentityBoth : Covariant, A, B](fa: F[A])(f: A => G[B]): G[Option[B]] =
```

Comment:
IdentityEither is useless for these case, we don't want to use IdentityEither.none in case of empty traversable^ because it 'none' behaves like failure 
So i requested IdentityBoth for any ;  may be there should be separate typeclass in hierarchy that produces G[Any]?